addons/website/static/src/snippets/s_popup/000.scss

```diff
@@ -68,7 +68,7 @@
             }
         }
         .s_popup_content {
-            z-index: 1;
+            z-index: $zindex-modal;
```

Comment:
Remind me why that change?

Reply:
To have the popup with a z-index higher than the header. Otherwise, if the popup is in top position, you cannot close it because it is behind the header

Reply:
Ok in fact I was confused because we now have a 001.scss for this snippet and the new version is always a standard BS modal structure so it should already have a naturally correct z-index :+1: It's ok to keep this as a fix for the old snippet version :) Just make sure the new 001 version works too :+1:

Reply:
it's working :+1: